src/fields-match.ts

```diff
@@ -0,0 +1,29 @@
+import { FieldValidationResult } from 'lc-form-validation';
+
+const defaultInvalidMessage = 'Fields do not match';
+
+const isString = (input: any) => typeof input === 'string' || input instanceof String;
+const getNestedKey = (nestedObj: any, path: string) =>
+  path.split('.').reduce((obj, key) => (obj && obj[key] !== undefined ? obj[key] : undefined), nestedObj);
+
+export const VALIDATION_TYPE = 'FIELDS_MATCH';
+
+export const validateFieldsMatch = (value: string, vm: any, key: string): FieldValidationResult => {
+  const fieldValidationResult: FieldValidationResult = new FieldValidationResult();
+  let fieldsAreEqual = false;
+
+  const nestedKeyResult = getNestedKey(vm, key);
+  if (nestedKeyResult === undefined) {
+    fieldValidationResult.errorMessage = 'The field pass by customParams is wrong';
```

Comment:
Here I would just skip the validation (no error, for the user is not validation), maybe add a console.log warning message indicadting something like:

```typescript
console.warn(`FieldMatch Validator: field id ${key} to compare not found`)
```

Reply:
Ok. Then I will update the test file with that.